CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # HEAD
 
+## Features
+* **assets:** Upgrade Inter font to version 4.1 (bedrock #15999)
+
+# 19.3.0
```

Comment:
```suggestion
```

We haven't published since the last update, so you can place this under the existing HEAD Features with karma to jasmine update